package/environments/base.js

```
@@ -1,116 +1,103 @@
+"use strict";
```

ESLint

[error] 'use strict' is unnecessary inside of modules.

(strict)